habitat-lab/habitat/tasks/nav/instance_image_nav_task.py

```diff
@@ -263,3 +263,10 @@ class InstanceImageNavigationTask(ObjectNavigationTask):
     image. Built on top of ObjectNavigationTask. Used to explicitly state a
     type of the task in config.
     """
+
+
+@registry.register_task(name="InstanceImageNav-v2")
+class InstanceImageNavigationTaskV2(InstanceImageNavigationTask):
```

Comment:
Is it really necessary ?

Reply:
The idea by @devendrachaplot was that having a separate version for the current task will allow us others to refer to this task with a simple name. Otherwise people will be calling it continuous control objectnav which is much more verbose.

Reply:
This class does not seem to enforce the continuous action space is used. It looks like it is the same class by another name. We will need to be consistent with how we use it.